src/services/config.js

```diff
@@ -146,6 +147,7 @@ class ConfigData {
     this.edge = configObj.edge;
     this.email = configObj.email || {};
     this.prompts = configObj.prompts || {};
+    this.profiles = configObj.profiles || null;
```

Comment:
No need to `|| null` since it will be `null/undefined` anyway.

Reply:
Thanks for pointing. Will update it accordingly.

Reply:
@onuzbee we shouldn't mark as `configObj.profiles || {}` because to differentiate the case where older CLI (profiles key itself not there in the config file) vs updated cli (profiles key is there but there are no profiles configured).